src/cpp/shared/gui/msg.cpp

```diff
@@ -31,3 +35,13 @@ void GUIBroadcastMessage(heroWindow* hwnd, int f, int c, void* p) {
 	evt.payload = p;
 	hwnd->BroadcastMessage(evt);
 }
+
+int GUIGetDropdownSelection(heroWindow* hwnd, void* p) {
+  tag_message evt;
+  evt.yCoordOrFieldID = 100;
+  evt.eventCode = INPUT_GUI_MESSAGE_CODE;
+  evt.xCoordOrKeycode = 55;
+  evt.payload = p;
```

Comment:
I want to note that I'm seeing 4 lines of code instead of just that one line. This behavior is similar to behavior in your other comments. Should I understand that you will be referring to the last line in situations like these from now on or is there something strange going on here? Perhaps maybe I'm not understanding something else.